site.yml

```diff
@@ -99,14 +99,6 @@
     - frontend
     - importer
 
-- hosts: all
-  roles:
-    - { role: upgrade-completion, when: "installation_mode == 'upgrade'" }
-  tags:
-    - backend
-    - frontend
-    - importer
-
 - hosts: frontends
```

Comment:
shouldn't the cleanup role run on all hosts, not just frontends?

Reply:
Partially yes, it cleans up all of these parts. These tags are a combination of all the tags associated with the tasks done prior to the cleanup role